sql/moz-fx-data-shared-prod/monitoring/shredder_progress/view.sql

```diff
@@ -42,12 +42,12 @@ WITH shredder AS (
 jobs AS (
   -- https://cloud.google.com/bigquery/docs/information-schema-jobs
   SELECT
-    *
+    * EXCEPT (query_info)
```

Comment:
I think this would do it (with the same for the other side of the union):
```suggestion
    project_id,
    job_id,
    creation_time,
    start_time,
    end_time,
    state,
    error_result,
    total_bytes_processed,
    total_slot_ms
```

Reply:
Double checked and updated it